### PR TITLE
Remember the features that are checked for the profiler

### LINKED ISSUE #110: Features do not stay checked properly
Steps to reproduce:
 * Open up add-on panel.
 * Uncheck "Stack walk"
 * Click "Apply (Restart Profiler)"
 * Close the panel
 * Open it again

Expected Results:
 * "Stack walk" remains unchecked

Actual Results:
 * "Stack walk" is now checked.

### PR DESCRIPTION
**Pull request by gregtatum**
Fixes #110

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -23,9 +23,8 @@ function renderState(state) {
     discreteLevelNotch.classList.toggle('inactive', !isActive);
   }
   const information = calculateInformation(state);
-  document.querySelector(
-    '.relevancy-level-fill'
-  ).style.width = `${information * 100}%`;
+  document.querySelector('.relevancy-level-fill').style.width = `${information *
```
Comment (gregtatum) [inline popup.js:26]: Prettier did this change.

File: popup.js [hunk 2]
```diff
@@ -195,19 +194,21 @@ document
...
-      background.adjustState({ features });
-      renderState(background.profilerState);
-    });
+async function setupFeatureCheckbox(featureName) {
+  const checkbox = document.querySelector(`.${featureName}-checkbox`);
+  const background = await getBackground();
+  checkbox.checked = background.profilerState.features[featureName];
+
+  checkbox.addEventListener('change', async e => {
+    const background = await getBackground();
```
Comment (mstange) [inline popup.js]: Could we just use the `background` variable from the outer scope?

**Review by mstange — APPROVED** [review]

File: popup.js (hunk 2 above)
Comment (gregtatum) [reply popup.js]: Yeah, I was being conservative since I didn't really know the behavior of the method under the hood.

Comment (mstange) [reply popup.js]: ok, I'm not entirely sure how it works either, so feel free to land what you have.

**Pull request merged**